hackclub.com.yaml

```diff
@@ -3051,3 +3051,7 @@ gps:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+wggs:
+  - ttl: 600
+    type: CNAME
+    value: https://luna5379.github.io.
```

Comment:
Hi, could you sort this alphabetically within the file?

Reply:
yes ill do that now

Reply:
done!